src/main/java/ch/njol/skript/util/FileUtils.java

```diff
@@ -60,7 +63,24 @@ public static String getBackupSuffix() {
 			return "" + backupFormat.format(System.currentTimeMillis());
 		}
 	}
-	
+
+	public static void backupPurge(final File f, int tokeep) throws IOException {
+		if (tokeep < 0)
+			throw new IOException("Called with invalid input");
```

Comment:
IllegalArgumentException (or use Preconditions) Also say why it's invalid.

Reply:
@sovdeeth Through Skript itself, it wont call this method if the `toKeep` is less than 0, This check is purely just incase anyone calls it through skript-reflect for example.

Reply:
That's why you should use IllegalArgumentException. It's an illegal argument, not an io error

Reply:
Yea, I added it